tests/test_0074-argsort-and-sort.py

```diff
@@ -7,8 +7,22 @@
 import pytest
 import numpy
 
+#import pickle
 import awkward1
 
+# def test_gordons_test():
+#     array = awkward1.Array([[149, 115, 73.5, 49.1, 44.1, 40.8], [151, 94.8]]).layout
+#     assert awkward1.to_list(array.argsort(1, True, False)) == [[5, 4, 3, 2, 1, 0], [1, 0]]
+#     with open('arg_sort_boom.pcl', 'rb') as f:
+#         data = pickle.load(f)
+#
+#         so = awkward1.argsort(data, axis=-1)
+#         assert awkward1.to_list(so) == [[5, 4, 3, 2, 1, 0], [1, 0]]
+#         assert awkward1.to_list(awkward1.sort(data, axis=-1)) == data[so]
+
+def test_keep_None_in_place_test():
+    assert awkward1.to_list(awkward1.argsort(awkward1.Array([[3, 2, 1], [], None, [4, 5]]), axis=1)) == [[2, 1, 0], [], None, [0, 1]]
```

Comment:
This looks good! Does it also work for `sort`?

Reply:
yes, I'm cleaning up the tests and have added one for sort.